libraries/rasclient/client_helper.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from requests import exceptions
+
+RESOURCE_ABSTRACTOR_ADDR = (
+    f"http://{os.environ.get('RESOURCE_ABSTRACTOR_URL')}:"
+    f"{os.environ.get('RESOURCE_ABSTRACTOR_PORT')}"
+)
+
+
+def make_request(method, api, **kwargs):
+    url = f"{RESOURCE_ABSTRACTOR_ADDR}{api}"
+    try:
+        response = method(url, **kwargs)
+        response.raise_for_status()
+        return response.json()
+    except exceptions.RequestException:
+        print(f"Calling {url} not successful.")
```

Comment:
Do we already have some kind of logger (that we could use here) that we could use here or should we really continue using prints for this? If I am not mistaken Luca mentioned that he will introduce something that will have the same "interface" as the python logging util, so maybe we could already use it here to avoid future heavier refactoring.

Reply:
I'll check out how luca is doing it. Perhaps we can tag him here to give us quick suggestions. Do you know his github handle, I am unable to tag/find him?

Reply:
I think he is not yet really part of the github project or so.
This is his account: https://github.com/TheDarkPyotr

Reply:
Not joined but reading here and there 😃 
For the logging approach at the moment the only approach I suggest is to use the standard `logging` library as the standardization process (*at the moment*) is a simple wrapper of the library but further discussion is needed. 
Next meeting surely will brought up the topic!